plugins/cassandra/cassandrawu.cpp

```diff
@@ -3126,6 +3153,8 @@ class CCasssandraWorkUnitFactory : public CWorkUnitFactory, implements ICassandr
             {
                 int major = versionInfo->getPropInt("@major", 0);
                 int minor = versionInfo->getPropInt("@minor", 0);
+                partitions = versionInfo->getPropInt("@numPartitions", DEFAULT_PARTITIONS);
```

Comment:
@richardkchapman are these values saved anywhere?  I don't think they're in the version mappings.